xrayutilities/materials/atom.py

```diff
@@ -64,6 +64,9 @@ def __init__(self, name, num):
         self.__weight = None
         self._dbcache = dict([(prop, []) for prop in ('f0', 'f1', 'f2', 'f')])
 
+    def swapbasename(self,name):
+        self.name = name
+
```

Comment:
Did you use that function in some other context? I am not sure if I find it very good to solve this like this. Since name is a simple string attribute one can set it directly. if other things are connected with a name change a property setter function should be used.

Reply:
Agree. 
Changing attributes shouldn't be done this way. (can't recall the motivation behind this).